pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Namespaces.java

```diff
@@ -143,6 +143,7 @@ public interface Namespaces {
      * @throws PulsarAdminException
      *             Unexpected error
      */
+    @Deprecated
```

Comment:
We don't need to deprecate the existing method?

Reply:
ok, fixed.